### ISSUE TITLE
Ignore files?

### ISSUE THREAD

**Issue by usandfriends** [summitappdevelopers/SummitTalks#2]
You uploaded node_modules. Do you want to create a `.gitgnore` to remove such files?

**Comment by usandfriends** [comment]
Note that if you are going to remove `node_modules` folders, you should write up a `package.json` file to help cloners install dependencies.

**Comment by Carpetfizz** [comment]
Yep! I'm deploying to OpenShift right now so I want to make sure `master` is working before I touch anything.
